dot/network/message.go

```diff
@@ -76,7 +76,7 @@ func (sd SyncDirection) String() string {
 // BlockRequestMessage is sent to request some blocks from a peer
 type BlockRequestMessage struct {
 	RequestedData byte
-	StartingBlock variadic.Uint64OrHash // first byte 0 = block hash (32 byte), first byte 1 = block number (int64)
+	StartingBlock variadic.Uint32OrHash // first byte 0 = block hash (32 byte), first byte 1 = block number (int64)
```

Comment:
I think comment needs to be updated? :thinking: 

```suggestion
	StartingBlock variadic.Uint32OrHash // first byte 0 = block hash (32 byte), first byte 1 = block number (uint32)
```

Reply:
updated